generators/generator-bot-command-list/README.md

```diff
@@ -1,42 +1,32 @@
 # @microsoft/generator-bot-command-list [![NPM version][npm-image]][npm-url]
 
-This template instantiates a bot built around a set of simple commands. Includes cards with message back actions for example commands "hello" and "help."
+This template creates a bot built around a set of simple commands. It includes cards with "message back" actions for the commands "hello" and "help."
```

Comment:
nit: it would be helpful to link back to the official docs for further explanation on messageBack activities. https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-actions#messageback